.github/workflows/haskell-ci-simple.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           - "8.10.7"
           - "9.0.2"
           - "9.2.7"
-          - "9.4.5"
+          - "9.4.4"
```

Comment:
Shouldn't 9.4.5 work with the latest haskell-setup?  @haskell-actions/setup@v2  (note migration to a new organization)